Survi.Prevention.Models/DataTransfertObjects/InspectionForDashboard.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace Survi.Prevention.Models.DataTransfertObjects
+{
+    public class InspectionForDashboard
+    {
+		public Guid Id { get; set; }
+        public Guid? IdRiskLevel { get; set; }
+        public string Address { get; set; }
+        public Guid? IdWebuserAssignedTo { get; set; }
+        public Guid IdBatch { get; set; }
+        public string BatchDescription { get; set; }
+        public DateTime VisitDate { get; set; }
+        public string VisitStatus { get; set; }
+        public Boolean VisitNote { get; set; }
```

Comment:
VisitNote and Anomaly are both booleans, so their name should show that : HasVisitNote, HasAnomaly.